automation/src/test/scala/org/broadinstitute/dsde/workbench/config/LeoAuthToken.scala

```diff
@@ -0,0 +1,53 @@
+package org.broadinstitute.dsde.workbench.config
+
+import com.google.api.client.googleapis.auth.oauth2.GoogleCredential
+import com.google.api.client.googleapis.javanet.GoogleNetHttpTransport
+import com.google.api.client.json.jackson2.JacksonFactory
+import org.broadinstitute.dsde.workbench.auth.AuthToken
+
+import scala.collection.JavaConverters._
+
+case class LeoAuthToken(value: String)
+
+case object LeoAuthToken extends AuthToken {
```

Comment:
I am curious how this class is different from `AuthToken`, `UserAuthToken`, `ServiceAccountAuthToken` classes in workbench-libs:

https://github.com/broadinstitute/workbench-libs/tree/2801-sam/serviceTest/src/test/scala/org/broadinstitute/dsde/workbench/auth

Could one of those classes be used instead?

Reply:
We've got a card already for the iteration of this - I can add refactoring this to it

Reply:
And I've added it.